format/Schema.fbs

```diff
@@ -171,6 +172,24 @@ table LargeUtf8 {
 table LargeBinary {
 }
 
+/// Same as Utf8, but string characters are delimited with a packed
+/// length/pointer instead of offsets.
+
```

Comment:
Not sure this is useful as the description just below is more precise.

Reply:
Oops, I thought I'd deleted this.